src/components/activities/ActivityIndex.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import ActivityCollection from './ActivityCollection'
+
+export default class ActivityIndex extends Component {
+
+  render() {
+    return(
+      <ActivityIndexWrapper>
+        <div>
+          <h3>Eerstvolgende Activiteit</h3>
+          <ActivityCollection count="0"/>
+          <p className="more"><a href="/vereniging/activiteiten">Meer activiteiten</a></p>
+        </div>
+      </ActivityIndexWrapper>
+    )
+  }
+}
+
+const ActivityIndexWrapper = styled.div`
+  @media (min-width: 990px) {
+    width: 80%; 
```

Comment:
Overall, I think we should look into `vw` and `vh` as dimensions, instead of percentages. Or `em`'s, but I am not too fond of percentages as these tend to fail in some cases (80% of what?). the other proposed are not relative to parent, but more 'concrete'